packages/contracts/contracts/optimistic-ethereum/OVM/execution/OVM_ExecutionManager.sol

```diff
@@ -1591,6 +1579,13 @@ contract OVM_ExecutionManager is iOVM_ExecutionManager, Lib_AddressResolver {
     uint256 _gasLimit,
     Lib_OVMCodec.QueueOrigin // _queueOrigin
   ) internal view returns (bool _valid) {
+    // Ensure that sufficient gas is provided to cover transaction.gasLimit.
+    // This includes 1/64 of the gas getting lost because of EIP-150 (lost twice--first
+    // going into EM, then going into the code contract).
+    if(gasleft() < 100000 + _gasLimit * 1032 / 1000) { // 1032/1000 = 1.032 = (64/63)^2 rounded up
```

Comment:
```suggestion
    // Ensure that sufficient gas is provided to cover transaction.gasLimit.
    // This includes 1/64 of the gas getting lost because of EIP-150.
    if(gasleft() < 100000 + _gasLimit * 64/63) { 
```